project_liberation/requirements.lock

```diff
@@ -0,0 +1,36 @@
+appdirs==1.4.3
```

Comment:
Please move both requirements to upper dir.

Reply:
You already changed it to pipenv ;)